### PR TITLE
Creates the minutes from the 2022-11-30 meeting

### PR DESCRIPTION
**Pull request by JesusSierralaya**

### CODE REVIEW

File: doc/minutes/2022-12-01_first_meeting.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title:  "First meeting YOS Award"
+date:   "30-11-2022"
+editor_options:
+  markdown:
+    canonical: false
+---
+
+**Attendees:** <names and surnames of attendees>
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: Delete wildcard `<names and surnames of attendees>`

File: doc/minutes/2022-12-01_first_meeting.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+-   Cristina Rodríguez-Prada
+
+-   Lea Francia
+
+-   Jesus Sierralaya
+
+**Scribe:** Jesus Sierralaya
+
+**Schedule:** 11:00 - 11:30 <!-- # Init time - End time -->
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: Delete wildcard `<!-- # Init time - End time -->`

File: doc/minutes/2022-12-01_first_meeting.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+# Action points
+
+## Project Proposals
+
+### Robert's proposal
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: "Roberto" I guess (but what @robertomediavilla is more comfortable with)

Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: These sections must go inside "Debate" (do not use new headings preferrably)
Also, it's not common practice to name the person that makes a proposal. We share accountability for the results as a whole team ;)

File: doc/minutes/2022-12-01_first_meeting.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+-   Cloud data management system and federated analysis
+
+-   Software Obiba Opal
+
+-   We already have a physical server
+
+### Lea's proposal
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: These sections must go inside "Debate" (do not use new headings preferrably)

File: doc/minutes/2022-12-01_first_meeting.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+-   Conducting a study from scratch to publication with GitHub
+
+### Daniel's proposal
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: These sections must go inside "Debate" (do not use new headings preferrably)

File: doc/minutes/2022-12-01_first_meeting.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+-   Similar to Lea's proposal
+
+-   We can use Synchros as expample
+
+#### Debate <!-- # Delete if unneeded -->
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: Delete wildcard `<!-- # Delete if unneeded -->`

File: doc/minutes/2022-12-01_first_meeting.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+-   What we're going to do with the money?
+
+    -   Pay a freelancer to maintain the server
+
+    -   General communication
+
+    -   Staff training
+
+#### Agreements <!-- # Delete if unneeded; delete/repeat each point as needed -->
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: Delete wildcard `<!-- # Delete if unneeded; delete/repeat each point as needed -->`

File: doc/minutes/2022-12-01_first_meeting.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+-   **Approved**:
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: Please make one "approved" point for each agreement. E.g.:

- **Approved**: Make a combination of both proposals
- **Approved**: Use the money for staff training

**Review by robertomediavilla — DISMISSED** [review]

File: doc/minutes/2022-12-01_first_meeting.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+### Daniel's proposal
+
+-   Similar to Lea's proposal
+
+-   We can use Synchros as expample
+
+#### Debate <!-- # Delete if unneeded -->
+
+-   What we're going to do with the money?
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: This is another action point (with heading level 2), not within "project proposals"

File: doc/minutes/2022-12-01_first_meeting.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+    -   Pay a freelancer to maintain the server
+
+    -   General communication
+
+    -   Staff training
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: These three points are the "Debate" within the "Prize assignation" action point

File: doc/minutes/2022-12-01_first_meeting.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+#### Agreements <!-- # Delete if unneeded; delete/repeat each point as needed -->
+
+-   **Approved**:
+
+-   Make a combination of both proposals
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: Of "the three proposals" (both applies to "two"). Also, the proposal goes a bit further I understand, including:

- Cloud data management system and federated analysis with Opal
- Reproducible research best practices with git/GitHub and literate computing

File: doc/minutes/2022-12-01_first_meeting.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+
+-   Use the money for staff training
+
+# **Task summary:**
```
Comment (DaniMori) [inline doc/minutes/2022-12-01_first_meeting.md]: Tasks in the "task summary" should also be within the action points. This here is a summary of the tasks previously stated in the action points.

**Review by DaniMori — CHANGES_REQUESTED** [review]
Nice work @JesusSierralaya . There are a few changes to be made to "meet a standard template", check them out.

**Comment by DaniMori** [comment]
Pull Request (PR) title: Please note, the title should state what this PR does, not what the branch has.

E.g., you want to say that it "Creates (or adds, merges...) the minutes from the 2022-11-30 meeting"

**Comment by robertomediavilla** [comment]
> Pull Request (PR) title: Please note, the title should state what this PR does, not what the branch has.
> 
> E.g., you want to say that it "Creates (or adds, merges...) the minutes from the 2022-11-30 meeting"

Wow that is super interesting. I always struggle with what to write when I commit. I should then always think something like "this commit does X and removes Y", right?

**Comment by DaniMori** [comment]
> Wow that is super interesting. I always struggle with what to write when I commit. I should then always think something like "this commit does X and removes Y", right?

Exactly! I'd advice to consider omitting "this commit" and state the predicate as a 3-rd person description of what the commit does; e.g. "fills in first version of the draft" or something like that

**Comment by DaniMori** [comment]
@JesusSierralaya , remember to update the minutes name to "only date"

File: doc/minutes/2022-12-01_first_meeting.md (hunk 6 above)
Comment (Cristrinaranjus) [reply doc/minutes/2022-12-01_first_meeting.md]: typo: expample

**Comment by robertomediavilla** [comment]
> > Wow that is super interesting. I always struggle with what to write when I commit. I should then always think something like "this commit does X and removes Y", right?
> 
> Exactly! I'd advice to consider omitting "this commit" and state the predicate as a 3-rd person description of what the commit does; e.g. "fills in first version of the draft" or something like that

Sure! Yes, that's what I meant. Thanks!

File: doc/minutes/2022-12-01_first_meeting.md (hunk 5 above)
Comment (Cristrinaranjus) [inline doc/minutes/2022-12-01_first_meeting.md]: I pointed out at the meeting that I agreed with both Rober's and Lea's proposals and that a combination of the two could then be made, along Dani's direction.

**Review by Cristrinaranjus — CHANGES_REQUESTED** [review]
Also agreeing with Dani's suggestions, I am including a comment about my participation in the meeting to be reflected in the same way :)

**Comment by JesusSierralaya** [comment]
> Pull Request (PR) title: Please note, the title should state what this PR does, not what the branch has.
>

File: doc/minutes/2022-12-01.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+    
+    -   We already have a physical server
+
+-   Reproducible research best practices with git/GitHub and literate computing
+
+-   We can use Synchros as example
+
+##   Prize assignation
+
+    -   Pay a freelancer to maintain the server
```
Comment (DaniMori) [inline doc/minutes/2022-12-01.md]: Indentation level "0" (i.e. left-aligned)

File: doc/minutes/2022-12-01.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+    -   General communication
+
+    -   Staff training
+
+#### Agreements 
+
+-   **Approved**: Make a combination of both proposals
```
Comment (DaniMori) [inline doc/minutes/2022-12-01.md]: This agreement goes in action point "Project Proposals"

File: doc/minutes/2022-12-01.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+
+-   **Approved**: Use the money for staff training
+
+## **Task summary:**
```
Comment (DaniMori) [inline doc/minutes/2022-12-01.md]: Include the tasks also in the corresponding action point (see template for format)

File: doc/minutes/2022-12-01.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+#### Debate 
+
+-   Cloud data management system and federated analysis with Opal
+    
+    -   We already have a physical server
+
+-   Reproducible research best practices with git/GitHub and literate computing
+
+-   We can use Synchros as example
+
```
Comment (DaniMori) [inline doc/minutes/2022-12-01.md]: Add level-4 header "Tasks" and include here the tasks with the format in the template

File: doc/minutes/2022-12-01.md [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+#### Agreements 
+
+-   **Approved**: Make a combination of both proposals
+
+-   **Approved**: Use the money for staff training
+
+## **Task summary:**
+
+-  (Daniel) Create repository
```
Comment (DaniMori) [inline doc/minutes/2022-12-01.md]: According to template, format here should be e.g.

```
## Daniel

- Create repository
```

File: doc/minutes/2022-12-01.md (hunk 16 above)
Comment (DaniMori) [reply doc/minutes/2022-12-01.md]: Mind the format in the template:

```
#### Tasks

- **Daniel**: Create repository
```

**Review by DaniMori — CHANGES_REQUESTED** [review]
Excellent job @JesusSierralaya !! Very concise and "to the point", no clutter, well done!

Requested changes are mainly about formatting, see if you can address them, should be straightforward ;)

File: doc/minutes/2022-12-01_first_meeting.md (hunk 5 above)
Comment (DaniMori) [reply doc/minutes/2022-12-01_first_meeting.md]: @Cristrinaranjus I think you may consider resolving this conversation now.

File: doc/minutes/2022-12-01.md [hunk 18]
```diff
@@ -0,0 +1,67 @@
+---
+title:  "First meeting YOS Award"
+date:   "30-11-2022"
+editor_options:
+  markdown:
+    canonical: false
+---
+
+**Attendees:**
```
Comment (DaniMori) [inline doc/minutes/2022-12-01.md]: Use "comma-separated" names, as agreed; see channel *CCOMS/YOS_Award*

File: doc/minutes/2022-12-01.md [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+
+# Action points
+
+## Project Proposals
+
+#### Debate 
+
+-   Cloud data management system and federated analysis with Opal
+    
+    -   We already have a physical server
```
Comment (DaniMori) [inline doc/minutes/2022-12-01.md]: Maybe add this in parentheses after the main bullet point?

File: doc/minutes/2022-12-01.md [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+-   **Approved**: Use the money for staff training
+
+#### Tasks
+
+- **Daniel**: Create repository
+
+- **Roberto**: Create the proposal outline
+
+##   Prize assignation
+
```
Comment (DaniMori) [inline doc/minutes/2022-12-01.md]: Add subheader "Debate"

File: doc/minutes/2022-12-01.md [hunk 21]
```diff
@@ -0,0 +1,67 @@
...
+-   Reproducible research best practices with git/GitHub and literate computing
+
+-   We can use Synchros as example
+
+#### Agreements 
+
+-   **Approved**: Make a combination of both proposals
+
+-   **Approved**: Use the money for staff training
```
Comment (DaniMori) [inline doc/minutes/2022-12-01.md]: This agreement should go in action point "Prize assignation"

**Review by DaniMori — CHANGES_REQUESTED** [review]
Excellent work Jesus!! Very good use of the template, and very concise. Please mind the three minor changes requested ;)

**Comment by DaniMori** [comment]
@JesusSierralaya, I forgot something else: Please correct the file name, it should be the date when the meeting took place.

**Review by Leafrancia — DISMISSED** [review]

**Review by DaniMori — CHANGES_REQUESTED** [review]
The changes I requested last time are still pending.

Also, mind the name of the file: It must be "2022-11-30.md", the day the meeting took place

**Review by JesusSierralaya — COMMENTED** [review]
I have made all proposed changes. Thank you!

**Review by Cristrinaranjus — DISMISSED** [review]

**Review by JesusSierralaya — COMMENTED** [review]
Solved in the commit https://github.com/CCOMS-UAM/yos-award/commit/38c7806df3f3d777aef834d4560f2b266eebb4bb

**Review by JesusSierralaya — COMMENTED** [review]
Solved in the commit https://github.com/CCOMS-UAM/yos-award/commit/38c7806df3f3d777aef834d4560f2b266eebb4bb

**Review by JesusSierralaya — COMMENTED** [review]
Solved in the commit https://github.com/CCOMS-UAM/yos-award/commit/38c7806df3f3d777aef834d4560f2b266eebb4bb

**Comment by JesusSierralaya** [comment]
> Solved in the commit https://github.com/CCOMS-UAM/yos-award/commit/38c7806df3f3d777aef834d4560f2b266eebb4bb

**Review by JesusSierralaya — COMMENTED** [review]
Solved in https://github.com/CCOMS-UAM/yos-award/commit/38c7806df3f3d777aef834d4560f2b266eebb4bb

**Review by DaniMori — APPROVED** [review]
That's perfect!!

**Comment by JesusSierralaya** [comment]
Hello @Cristrinaranjus and @robertomediavilla please could you review this PR again? Thank you!

**Review by Cristrinaranjus — APPROVED** [review]

**Review by robertomediavilla — APPROVED** [review]

**Pull request merged**